LongestPalindromicSubstring.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
+    def longestPalindrome(self, s: str) -> str:
+        L = R = l = r = 0
+
+        while r < len(s):
+            if l == r:
+                # Looking for the 'root' of palindrome,
+                # either odd or even number of same characters
+                while r + 1 < len(s) and s[l] == s[r+1]:
+                    r += 1
+                    if l > 0 and s[l - 1] == s[r]:
+                        l -= 1
```

Comment:
Wouldn't it be better to stip out to a function like `findPalindrome`?

Reply:
Logically it has not enough meaning. Just finding center moving to a function, idk( You think it's worth it?

Reply:
It's up to you, I'd do that but it's not necessary for sure.